translations/README-en.md

```diff
@@ -0,0 +1,218 @@
+![Logotype](../logo.png)
+
+> Team Lead is a ❄️, because in each company they are unique and distinct.
+
+- [Roadmap](#roadmap)
+- [Telegram chat TL Bootcamp (RU)](https://tlinks.run/tlbootcamp)
+
+# What is it about?
+
+The concepts of "Team Lead", "Tech Lead", "Tech Manager" and others are very stretchy. Companies solve different tasks, these roles have different responsibilities. Besides, different leadership styles, differences in personal skills, and much more are put on the picture. But isn't that what happens to the developers? Different companies, different tasks, again, different nature of tasks and responsibilities.
+
+We interviewed several dozen large and small companies, found out what they expect from Team Leads and how they interact with them. As a result, we were able to form a single model of basic teamwork competencies that covers all the cases studied.
+
+This model can be used in any way you like - to come up with your development plan, to create job descriptions in companies, to write a job description or to conduct interviews. Keep in mind it is likely you don't need all the branches of potential development - and this is fine.
+
+> 🚨 **Important:** Depending on the company, the Team Lead needs to work on different branches and responsibilities. We don't think that the goal should be to develop all of the areas in the roadmap. It's important to focus on the problems and needs of your workplace, first of all.
+
+The work is still in progress, so feel free to publish an Issue, drop your pull request and join the discussion [in Telegram chat, RU](https://tlinks.run/tlbootcamp).
+
+# How to use it?
+## 🏫 For a company
+1. Download the [MindMap](roadmap.mm) with the full Team Lead model.
```

Comment:
```suggestion
1. Download the [MindMap](roadmap.mm) with the Team Lead model.
```

Reply:
Here, the "full" word has its special meaning, I don't think it should be removed.

Reply:
It's not clear what is special in this meaning (do we have a short Team Lead model?) but I don't mind to leave it as-is.

Reply:
Yep, we have subsets of the model, in "examples"

Reply:
Let's see if there are some other opinions. As a variant, we can change "full" to "complete"